operator-common/src/main/java/io/strimzi/operator/common/operator/resource/RoleBindingOperator.java

```diff
@@ -4,89 +4,35 @@
  */
 package io.strimzi.operator.common.operator.resource;
 
-import io.fabric8.kubernetes.api.model.OwnerReference;
+import io.fabric8.kubernetes.api.model.rbac.DoneableKubernetesRoleBinding;
+import io.fabric8.kubernetes.api.model.rbac.KubernetesRoleBinding;
+import io.fabric8.kubernetes.api.model.rbac.KubernetesRoleBindingList;
 import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.dsl.MixedOperation;
+import io.fabric8.kubernetes.client.dsl.Resource;
 import io.vertx.core.Future;
 import io.vertx.core.Vertx;
 
-/**
- * This class is a temporary work-around for the fact that Fabric8 doesn't
- * yet support an API for manipulating Kubernetes ClusterRoleBindings
- * @deprecated This can be removed once support for ClusterRoles and ClusterRoleBindings is in Fabric8.
- */
-@Deprecated
-public class RoleBindingOperator extends WorkaroundRbacOperator<RoleBindingOperator.RoleBinding> {
 
-    public static final String API_VERSION = "v1beta1";
-    public static final String GROUP = "rbac.authorization.k8s.io";
+public class RoleBindingOperator extends AbstractNonNamespacedResourceOperator<KubernetesClient, KubernetesRoleBinding, KubernetesRoleBindingList, DoneableKubernetesRoleBinding, Resource<KubernetesRoleBinding, DoneableKubernetesRoleBinding>> {
```

Comment:
RoleBindings are namespaces ... so this should extend the namespaced abstract operator.